tests/forte/data/readers/ag_news_reader_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for ag_news_reader.
+"""
+import os
+import unittest
+from typing import Dict
+
+from forte.pipeline import Pipeline
+from ft.onto.ag_news import Title, Description
+from ft.onto.base_ontology import Document
+
+from forte.data.readers import AGNewsReader
+from forte.data.data_pack import DataPack
+
+
+class AGNewsReaderTest(unittest.TestCase):
+
+    def setUp(self):
+        self.pipeline = Pipeline()
+
+        self.pipeline.set_reader(AGNewsReader())
+        self.pipeline.initialize()
+
+        self.sample_file: str = os.path.abspath(os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            *([os.path.pardir] * 4),
+            'data_samples/ag_news/sample.csv'))
+
+        self.expected_content: Dict[int, str] = {}
+        with open(self.sample_file, 'r') as file:
```

Comment:
If you simply one sample here, sometimes it is easier to put the expected text in raw string.  This can help avoid you making the same mistake in the reader and the test case.

Reply:
Thanks for the suggestion. But here I have multiple samples in one file. Since the expected content is generated differently from the data reader, it's not likely to make the same mistake.

Reply:
OK, I don't think this is a crucial point to fix anyway.